plugins/ruby-foreman-cockpit/debian/changelog

```diff
@@ -1,3 +1,9 @@
+ruby-foreman-cockpit (2.0.1-1) stable; urgency=low
+
+  * Update release to 2.0.1
+
+ -- Daniel Lobato Garcia <elobatocs@gmail.com>  Sun, Apr 10 2016 12:50:53 +0200
```

Comment:
Note that this date format's invalid and causes build failures. @mmoll has kindly fixed it in be0040a, but be aware for the future.  Use the standard deb tooling or scripts/* please.